libgdf/src/tests/arrow/arrow-test.cu

```diff
@@ -0,0 +1,92 @@
+#include "gtest/gtest.h"
+#include "gdf/cffi/types.h"
+#include "gdf/arrow.hpp"
+#include <arrow/api.h>
+#include <thrust/device_vector.h>
+#include <arrow/gpu/cuda_api.h>
+
+using namespace arrow;
+using namespace arrow::gpu;
+
+TEST(ArrowTests, ArrowToGDF){
+  gdf_column inputCol, outputCol, outputCol2;
+  std::vector<int32_t> inputData = {
+    17696,
+    17697,
+  };
+  inputCol.dtype = GDF_INT32;
+  inputCol.size = 2;
+  inputCol.valid = NULL;
+  inputCol.null_count = 0;
+  thrust::device_vector<int32_t> inputDataDev(inputData);
```

Comment:
Technically, this should be allocated using RMM. See https://github.com/rapidsai/cudf/blob/master/cpp/src/rmm/thrust_rmm_allocator.h